packages/framework/src/Form/Admin/Order/OrderFormType.php

```diff
@@ -135,6 +143,15 @@ private function createBasicInformationGroup(FormBuilderInterface $builder, Orde
         $builderBasicInformationGroup = $builder->create('basicInformationGroup', GroupType::class, [
             'label' => 'Basic information',
         ]);
+        $withdrawalRequest = $this->withdrawalRequestFacade->findByOrder($order);
```

Comment:
nitpick

I have a small concern for this method being called twice (maybe more, not sure how the computing validation group works exactly). 
Maybe we could set an option in configureDefaults and then get the withdrawalRequest from the option here and in the validation group callback?